Resources/sql/cleandata.sql

```diff
@@ -1,15 +1,21 @@
 INSERT INTO `ezcontentclass` VALUES (1,'<name>',1435924826,14,42,'landing_page',2,1,2,1435924826,14,'60c03e9758465eb69d56b3afb6adf18e','a:1:{s:6:\"eng-GB\";s:0:\"\";}','a:1:{s:6:\"eng-GB\";s:12:\"Landing page\";}',2,0,'',0);
+INSERT INTO `ezcontentclass` VALUES ('1', '<title>', '1537166773', '14', '43', 'form', '2', '0', '2', '1537166834', '14', '6f7f21df775a33c1e4bbc76b48c38476', 'a:0:{}', 'a:1:{s:6:\"eng-GB\";s:4:\"Form\";}', '2', '0', '', '0');
```

Comment:
please remove extra spaces after `,`, change strings to int according to first line/mysql field types